spras/dataset.py

```diff
@@ -89,6 +97,13 @@ def load_files_from_dict(self, dataset_dict):
                 "Direction",
             ]
 
+            weight_series = pd.Series(self.interactome.Weight)
+            weight_series_out_range = ~weight_series.between(0, 1)
```

Comment:
@agitter @annaritz Since a weight could be negative, should we allow that as well?

Reply:
Forgot to respond to this - wasn't there an issue open that established they can't be negative? #237

Reply:
In general, some graph algorithms do allow for negative weights. I forgot about this in #237.

Reply:
Yeah, let's ignore negative weights for now. If needed, we can add support for them in the #302 PR or address it in a separate PR after #302 merges, since that one already introduces a lot of new code. I suspect we won’t need to handle negative weights for the benchmarking datasets (and I'm assuming the people that use SPRAS right probably don't deal with negative weights), so in my ming it’s low priority. However, to make sure we don’t lose track of it, we should create a separate issue to revisit this later in case it becomes necessary.